.gitlab-ci/docker-ros.yml

```diff
@@ -47,19 +49,26 @@ variables:
 
   _RUN_IMAGE:             ${IMAGE_NAME}:${IMAGE_TAG}
   _DEV_IMAGE:             ${DEV_IMAGE_NAME}:${DEV_IMAGE_TAG}
+  _SLIM_IMAGE:            ${SLIM_IMAGE_NAME}:${SLIM_IMAGE_TAG}
 
   _IMAGE_DEV_CI:          ${_DEV_IMAGE}_${CI_COMMIT_REF_SLUG}_ci
   _IMAGE_RUN_CI:          ${_RUN_IMAGE}_${CI_COMMIT_REF_SLUG}_ci
+  _IMAGE_SLIM_CI:         ${_SLIM_IMAGE}_${CI_COMMIT_REF_SLUG}_ci
   _IMAGE_DEV_CI_AMD64:    ${_IMAGE_DEV_CI}-amd64
   _IMAGE_DEV_CI_ARM64:    ${_IMAGE_DEV_CI}-arm64
   _IMAGE_RUN_CI_AMD64:    ${_IMAGE_RUN_CI}-amd64
   _IMAGE_RUN_CI_ARM64:    ${_IMAGE_RUN_CI}-arm64
-  _IMAGE_DEV_LATEST:      ${DEV_IMAGE_NAME}:latest-dev
+  _IMAGE_SLIM_CI_AMD64:   ${_IMAGE_SLIM_CI}-amd64
+  _IMAGE_SLIM_CI_ARM64:   ${_IMAGE_SLIM_CI}-arm64
+  _IMAGE_DEV_LATEST:      ${DEV_IMAGE_NAME}:latest-dev # TODO: is this a bug here that DEV_IMAGE_TAG is not used?
```

Comment:
yes, seems so

Reply:
Actually not a bug, I think. `_IMAGE_DEV_LATEST` refers to the dev image that is pushed in the case that `push-as-latest` is active, a flag to also push `latest` even though a different tag is normally set.

We offer variables to control run tag and dev tag separately, but not for only controlling the `-dev` suffix. If we did, we could also consider using it here, but we don't. So I think we can keep it as-is.